app/views/private_api/lineages.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.array! @lineages do |lineage|
```

Comment:
I don't really see why we can't use the lineages that come with the `pept2data` call here?

Reply:
I created a seperate end-point for lineages so that lineages can still be requested when only taxa count tables are given. It is true however that,  when working with (Processed)PeptideContainers, the lineages returned from `pept2data` should be loaded directly into the NCBITaxonomy Ontology to minimize data exchange.